app/connectors/itsastatus/OptOutUpdateRequestModel.scala

```diff
@@ -21,9 +21,9 @@ import play.mvc.Http.Status.{INTERNAL_SERVER_ERROR, NO_CONTENT}
 
 object OptOutUpdateRequestModel {
 
-  val optOutUpdateReason = 10
+  val optOutUpdateReason: String = "10"
 
-  case class OptOutUpdateRequest(taxYear: String, updateReason: Int)
+  case class OptOutUpdateRequest(taxYear: String, updateReason: String)
```

Comment:
thanks James - do all the tests pass with this change?
We should have json reads/writes tests on all request models that would fail with the above change.

Reply:
Can we get that on another PR @joesheridan ?